docs/Deploy/aws_ec2.mdx

```diff
@@ -0,0 +1,160 @@
+---
+title: "Deploying to AWS EC2"
+sidebar_label: "AWS EC2"
+sidebar_position: 2
+---
+
+# Deploying to Amazon EC2
+
+Before you get started, Conduit should not be listening on any ports that are accessible to the public Internet. In this guide will we'll set up a Conduit instance that can be reached through the public Internet via SSH. We'll start at the top with a fresh AWS Account. In the end, the topology will look like this:
+
+![AWS EC2 Topology](/images/guides/aws-ec2-deployment.png)
```

Comment:
The build logs currently error with:

```shell
01:32:05.220 | Error: Image static/images/guides/aws-ec2-deployment.png used in docs/Deploy/aws_ec2.mdx not found.
-- | --
01:32:05.221 | at async Promise.all (index 0)
01:32:05.221 | [ERROR] Client bundle compiled with errors therefore further build is impossible.
01:32:05.263 | error Command failed with exit code 1.
01:32:05.264 | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
01:32:05.278 | Failed: build command exited with code: 1
01:32:06.265 | Failed: an internal error occurred
```

and updating the url should resolve the issue

Reply:
I added another commit to update the image url to the one you already added in the PR, but feel free to edit!

![Screen Shot 2022-07-21 at 12 53 19 PM](https://user-images.githubusercontent.com/8811742/180197569-4b2f0e0e-91da-4fd3-9086-64008cb70691.png)